src/wallet/wallet.cpp

```diff
@@ -1530,7 +1530,6 @@ void CWalletTx::GetAmounts(std::list<COutputEntry>& listReceived,
 int64_t CWallet::RescanFromTime(int64_t startTime, bool update)
 {
     AssertLockHeld(cs_main);
-    AssertLockHeld(cs_wallet);
```

Comment:
This is changed in #11281.

Reply:
Yes, #11281 resolves this. Thanks for pointing that out!